### DISCUSSION TITLE
Helix equivalent of vim comma

### DISCUSSION THREAD

**Discussion by musjj** [helix-editor/helix#8495] [Ideas]
There's the `repeat_last_motion` key, but what's the inverse? In vim, you can reverse the last motion by pressing `,`. 

So after, let's say pressing `fx`, you can press `,` to perform the equivalent of `Fx`. Is there an equivalent of this in Helix?

**Comment by BrianShTsoi** [comment]
yea I was wondering about the same thing. The reverse last motion key makes ```f```, ```t``` and ```A-.``` a lot more forgiving. Often times I would ```repeat_last_motion``` one too many time and want to go back
